src/dtos/monitor-span.dto.ts

```diff
@@ -109,12 +117,29 @@ export class MonitorSpanBaseDTO {
     example: propertyMetadata.monitorSpanDTOMpfValue.example,
     name: propertyMetadata.monitorSpanDTOMpfValue.fieldLabels.value,
   })
-  @IsOptional()
   @IsAtMostDigits(10, {
     message: (args: ValidationArguments) => {
       return `${args.property} [SPAN-FATAL-A] The value : ${args.value} for ${args.property} must be 10 digits or less`;
```

Comment:
use `CheckCatalogService.formatMessage` and remove the error code since those are not valid.